src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java

```diff
@@ -116,17 +116,32 @@ public void insideFolder() throws Exception {
         public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {}
     }
     private static abstract class MockNavigatorDescriptor extends SCMNavigatorDescriptor {
+
         MockNavigatorDescriptor() {
             super(MockNavigator.class);
         }
+
         @Override
         public SCMNavigator newInstance(String name) {
             return new MockNavigator();
         }
+
         @Override
         public String getDisplayName() {
             return "MockNavigator";
         }
+
+        public String getDescription() {
+            return "MockNavigator description";
+        }
+
+        public String getCategoryId() {
+            return "MockNavigator categoryId";
+        }
+
+        public String getIconFilePathPattern() {
+            return "MockNavigator iconfilepathpattern";
+        }
```

Comment:
Should not be necessary since these methods should not have been made `abstract`.